README.md

```diff
@@ -183,6 +183,69 @@ NB: It's also possible to run qdl from the host while the baord is not connected
 
 Want to join in the development? Changes welcome! See [CONTRIBUTING.md file](CONTRIBUTING.md) for step by step instructions.
 
+### Test an image locally with qemu
+
+You can boot an image locally with qemu as follows:
+
+1. As above under "Usage", build the disk image from the root filesystem
+   tarball if you haven't done this already:
+    ```bash
+    debos debos-recipes/qualcomm-linux-debian-image.yaml
+    ```
+
+1. Run qemu as follows:
+    ```bash
+    # SCSI is required to present a device with a matching 4096 sector size
+    # inside the VM
+    qemu-system-aarch64 -cpu cortex-a57 -m 2048 -M virt -nographic \
+        -device virtio-scsi-pci,id=scsi1 \
+        -device scsi-hd,bus=scsi1.0,drive=disk1,physical_block_size=4096,logical_block_size=4096 \
```

Comment:
Oh that's cool! Makes me think we should (separately) make a test to check the SD card/eMMC images boot too (with the normal 512 bytes sector size)!